floodsub.go

```diff
@@ -252,8 +252,9 @@ func (p *PubSub) announce(topic string, sub bool) {
 	}
 }
 
-// notifySubs sends a given message to all corresponding subscribbers.
-// Only called from processLoop.
+// notifySubs sends a given message to all corresponding subscribers.
+// A topic must have a subscription(s), otherwise notifySubs fails.
```

Comment:
please don't add this comment, it's incorrect.

Reply:
Done in 27e81ba